front/app/api/graph_data_units/responseTypes.ts

```diff
@@ -49,21 +49,26 @@ export type GroupedAnswer = Answer & {
   groups: { group: string | null; count: number }[];
 };
 
+export type SurveyQuestionMultilocAnswer = {
+  title_multiloc: Multiloc;
+  image?: ImageSizes;
+};
+
 export type SurveyQuestionMultilocs = {
```

Comment:
This one is also the same as `AnswerMultilocs` type in `api/survey_results/types`. Could be de-duplicated too.